python/semantic_kernel/core_plugins/sessions_python_tool/sessions_python_plugin.py

```diff
@@ -91,6 +91,19 @@ def _sanitize_input(self, code: str) -> str:
         code = re.sub(r"^(\s|`)*(?i:python)?\s*", "", code)
         # Removes whitespace & ` from end
         return re.sub(r"(\s|`)*$", "", code)
+    
+    def _construct_remote_file_path(self, remote_file_path: str) -> str:
+        """Construct the remote file path.
+
+        Args:
+            remote_file_path (str): The remote file path.
+
+        Returns:
+            str: The remote file path.
+        """
+        if not remote_file_path.startswith("/mnt/data/"):
+            remote_file_path = f"/mnt/data/{remote_file_path}"
```

Comment:
Should we make this path OS-aware? I don't think it would be the same in Windows :)

Reply:
Good question. This is the ACA specified/managed path - this is what their docs say all files get uploaded to.

Reply:
Oh well :)